crates/common_enums/src/enums.rs

```diff
@@ -2546,6 +2549,26 @@ impl AuthenticationConnectors {
     }
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    Hash,
+    PartialEq,
+    serde::Serialize,
+    serde::Deserialize,
+    strum::Display,
+    strum::EnumString,
+    ToSchema,
+)]
+#[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
+pub enum TaxationConnectors {
+    Taxjar,
+    Avalara,
```

Comment:
Only add 1 which we are integrating.
```suggestion
    Taxjar,
```